controllers/component_build_controller.go

```diff
@@ -216,21 +218,25 @@ func (r *ComponentBuildReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		}
 
 		// Manage merge request for Pipelines as Code configuration
-		bundle := gitopsprepare.PrepareGitopsConfig(ctx, r.NonCachingClient, component).BuildBundle
-		mrUrl, err := ConfigureRepositoryForPaC(component, pacSecret.Data, webhookTargetUrl, webhookSecretString, bundle)
-		if err != nil {
-			log.Error(err, "failed to setup repository for Pipelines as Code")
-			r.EventRecorder.Event(&component, "Warning", "ErrorConfiguringPaCForComponentRepository", err.Error())
-			return ctrl.Result{}, err
-		}
-		var mrMessage string
-		if mrUrl != "" {
-			mrMessage = fmt.Sprintf("Pipelines as Code configuration merge request: %s", mrUrl)
+		if val, ok := component.Annotations[skipPacResourceGenerationAnnotation]; ok && val == "1" {
+			r.EventRecorder.Event(&component, "Normal", "PipelinesAsCodeConfiguration", "Skipping the generation of .tekton resources")
 		} else {
-			mrMessage = "Pipelines as Code configuration is up to date"
+			bundle := gitopsprepare.PrepareGitopsConfig(ctx, r.NonCachingClient, component).BuildBundle
+			mrUrl, err := ConfigureRepositoryForPaC(component, pacSecret.Data, webhookTargetUrl, webhookSecretString, bundle)
+			if err != nil {
+				log.Error(err, "failed to setup repository for Pipelines as Code")
```

Comment:
I've noticed recently in jvm-build-service that with our default level of logging, controller runtime will log the generation of the event in the pod log as well; so you'll end up with essentially duplicate logs with this one ^^ and the event below.

Either in a manual bringup, or investigating the build-service logs collected in the artifacts of the e2e test, you should confirm if that is the case with build-service, and if so, just go with the event here.

@mmorhun FYI

Reply:
Yes, I can confirm, that events are logged as well and we have duplicates in logs (my mistake before).